bot/kodiak/test_evaluation.py

```diff
@@ -3242,15 +3255,17 @@ def test_get_merge_body_include_pull_request_author_user(
             merge=Merge(
                 method=MergeMethod.squash,
                 message=MergeMessage(
-                    body=MergeBodyStyle.empty, include_pull_request_author=True
+                    body=MergeBodyStyle.pull_request_body,
+                    include_pull_request_author=True,
                 ),
             ),
         ),
         pull_request,
+        commit_authors=[],
     )
     expected = MergeBody(
         merge_method="squash",
-        commit_message="\n\nCo-authored-by: Barry Berkman <828352+barry@users.noreply.github.com>",
+        commit_message="hello world\n\nCo-authored-by: Barry Berkman <828352+barry@users.noreply.github.com>",
```

Comment:
hmm why wasn't `hello world` in here before?

Reply:
I changed the test from `MergeBodyStyle.empty` to `MergeBodyStyle.pull_request_body`.